spec/api-tray-spec.js

```diff
@@ -2,18 +2,18 @@ const {remote} = require('electron')
 const {Menu, Tray, nativeImage} = remote
 
 describe('tray module', () => {
-  describe('tray.setContextMenu', () => {
-    let tray
+  let tray
 
-    beforeEach(() => {
-      tray = new Tray(nativeImage.createEmpty())
-    })
+  beforeEach(() => {
+    tray = new Tray(nativeImage.createEmpty())
```

Comment:
Can we have a test on Mac creating a `Tray` without an icon?

Reply:
this is how you do it without the icon, it does not accept `null`